x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -111,7 +111,7 @@ export function AddDataView() {
                 {`helm install --create-namespace -n=universal-profiling universal-profiling-agent \\
 --set "projectID=1,secretToken=${secretToken}" \\
 --set "collectionAgentHostPort=${collectionAgentHost}" \\
---set "version=${stackVersion}" \\
+--set "image.tag=${stackVersion}" \\
```

Comment:
We can remove setting the image tag entirely, because `image.tag` will default to the same chart version applied with L115 (`--version`).

Reply:
removed `image.tag` with https://github.com/elastic/kibana/pull/174876/commits/f03f67f0ebbecfd39052435af02a8fcafa5c6eb5